docker-compose.yml

```diff
@@ -10,74 +10,74 @@ services:
     image: quantil-db
     environment:
       # specifies user:pw for each container
-      POSTGRES_USERS: 'planqk:planqk|nisq:nisq|patternpedia:patternpedia|qprov:qprov'
+      POSTGRES_USERS: 'planqk:planqk|nisq:nisq|patternatlas:patternatlas|qprov:qprov'
       # specifies db name for each container
-      POSTGRES_DATABASES: 'planqk:planqk|nisq:nisq|patternpedia:patternpedia|qprov:qprov'
+      POSTGRES_DATABASES: 'planqk:planqk|nisq:nisq|patternatlas:patternatlas|qprov:qprov'
       ATLAS_DB: planqk
       NISQ_DB: nisq
-      PATTERNPEDIA_DB: patternpedia
+      PATTERNPEDIA_DB: patternatlas
       QPROV_DB: qprov
     #volumes:
       # use this volume to save the postgres data
       # - ./postgres-data:/var/lib/postgresql/data
     ports:
-      - '5432:5432'
+      - '5060:5060'
     networks:
       - default
   qc-atlas-ui:
-    image: planqk/qc-atlas-ui:v2.0.0
+    image: planqk/qc-atlas-ui:v3.0.0
     depends_on:
       - config-server
     environment:
       # because the requests are performed from the browser and not
       # from the container itself we have to specify localhost here
       # and **not** the container name
       QC_ATLAS_HOST_NAME: localhost
-      QC_ATLAS_PORT: 8080
+      QC_ATLAS_PORT: 6626
       PATTERN_ATLAS_HOST_NAME: localhost
-      PATTERN_ATLAS_PORT: 8082
+      PATTERN_ATLAS_PORT: 1977
       NISQ_ANALYZER_HOST_NAME: localhost
-      NISQ_ANALYZER_PORT: 8081
+      NISQ_ANALYZER_PORT: 5010
       QPROV_HOST_NAME: localhost
-      QPROV_PORT: 8084
+      QPROV_PORT: 5020
     ports:
       - '80:80'
     volumes:
       - ./config:/opt/init-config.d
     networks:
       - default
   qc-atlas:
-    image: planqk/atlas:v2.0.0
+    image: planqk/atlas:v3.1.0
```

Comment:
don't you mean qc-atlas-ui?

Reply:
@manuwei is it also possible to rename patternpedia in the db folder, e.g. in the setup-patternatlas.sh to completely remove the name?

Reply:
Yes, we should do that